MIT.md

```diff
@@ -0,0 +1,10 @@
+## Copyright 2021, [YOUR NAME]
+
+###### Please delete this line and the next one
+###### APP TYPE can be a webpage/website, a web app, a software and so on
```

Comment:
- [ ] [OPTIONAL] Be kind to not keep default contents in your `MIT.md` file. To make your repository more professional you should remove obvious comments and update your file.